documentation/providers/realtimeregister.md

```diff
@@ -26,7 +26,8 @@ If premium is set to "1", you will only be able to update zones using Premium DN
 will only be able to update zones using Basic DNS.
 
 **Important Notes**:
-* Anyone with access to this `creds.json` file will have *full* access to your RTR account and will be able to transfer or delete your domains
+* It is recommended to create a 'DNScontrol' user in your account settings with limited permissions, otherwise anyone with
```

Comment:
What would you advise? What limited permissions are there? Could you help someone else with this?

Reply:
```suggestion
* It is recommended to create a 'DNSControl' user in your account settings with limited permissions, otherwise anyone with
```

_https://docs.dnscontrol.org/developer-info/styleguide-doc#capitalization-matters_

Reply:
I provided some extra information.

Reply:
Thanks for the extra explanation.